src/cocotb/regression.py

```diff
@@ -126,8 +126,8 @@ class Test:
             Units of ``timeout_time``, accepts any units that :class:`~cocotb.triggers.Timer` does.
 
         expect_fail:
-            If ``True`` and the test fails a functional check via an ``assert`` statement, :pytest:class:`pytest.raises`,
-            :pytest:class:`pytest.warns`, or :pytest:class:`pytest.deprecated_call`, the test is considered to have passed.
+            If ``True`` and the test fails a functional check via an ``assert`` statement, :class:`pytest.raises`,
+            :class:`pytest.warns`, or :class:`pytest.deprecated_call`, the test is considered to have passed.
```

Comment:
The old code was supposed to use the pytest intersphinx setup from https://github.com/cocotb/cocotb/blob/e8c61232559d79787c5c4b306988608d9b742211/docs/source/conf.py#L56 which somehow doesn't produce a link (anymore?), but the change doesn't produce a link either.

Reply:
Hm. I thought I got it to produce a link again, but apparently not. I tried some other variants, but no luck. I'll leave it as-is for now, it at least doesn't produce a warning any more.